ui/AssessmentManager.py

```diff
@@ -23,23 +27,25 @@ def on_pre_enter(self, *args):
             self.user = ids.user
             self.module_number = ids.module_number
             self.assessment = ids.assessment
-            self.   app.title = "Health Friend [Assessment]  ::  " + self.user['first_name'] + " " + self.user['last_name']
+            self.   app.title = "Health Friend [Assessment]  ::  " + \
+                self.user['first_name'] + " " + self.user['last_name']
         else:
             self.app.title = "Health Friend [Assessment]  ::  EWH"
         self._load(self.module_number)
 
-
     def _load(self, module_number: int):
         print('loading', module_number)
-        with open() as file:
-            data = json.loads(file)
+        with open('assets/json/questions0.json') as file:
```

Comment:
can you create this string by using the passed in module_number instead of hard coding questions0.json